src/components/handbook-navigation/index.js

```diff
@@ -1,19 +1,38 @@
-import React from "react";
+import React, { useState } from "react";
 import { HiOutlineChevronLeft } from "react-icons/hi";
 import { Link } from "gatsby";
 import TOCWrapper from "./toc.style";
+import { IoMdClose, IoIosArrowDropdownCircle } from "react-icons/io";
 
 const TOC = () => {
+  const [expand, setExpand] = useState(false);
   return (
     <TOCWrapper>
       <div className="go-back">
         <Link to="/community/handbook">
           <HiOutlineChevronLeft />
           <h4>Table of Content</h4>
         </Link>
+        <div className="toc-toggle-btn">
+          {expand ? (
+            <IoMdClose
+              className="mobile-menu-icon toc-menu-icon open"
```

Comment:
Why 2 classes?

Reply:
The `toc-toggle-btn` is used to hide menu button for large screens. I think we can add those properties to `toc-menu-icon`.

Reply:
That's fine. But, what's the use of `mobile-menu-icon`, I doubt that is needed?

Reply:
Yes, We can remove it. Sorry, I took reference to the original mobile nav. I'll fix it immediately.

Reply:
I thought the classes were adding some properties but I guess it wasn't the case. Can you check now?